perf/auto-qual-test/deploy_latest_daily.sh

```diff
@@ -0,0 +1,9 @@
+git clone https://github.com/istio/tools
+cd tools/perf/istio-install
+helm init --client-only
+TARGET_VERSION=$1
+LATEST_BUILD=$(curl -L https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/release-$TARGET_VERSION-latest.txt)
```

Comment:
Are you only interested in release branches, or master as well? For master the name is master-latest.txt

Reply:
My priority is to get 1.1 working.  Will focus on master at a later point.